examples/arc/src/main.rs

```diff
@@ -2,7 +2,7 @@ use std::{f32::consts::PI, time::Instant};
 
 use iced::executor;
 use iced::widget::canvas::{
-    self, Cache, Canvas, Cursor, Geometry, Path, Stroke,
+    self, Cache, Canvas, Cursor, Geometry, Path, Stroke, StrokeStyle,
```

Comment:
Probably worth exposing a `stroke` module and simply define `Style` in there. E.g.

```rust
use widget::canvas::stroke::{self, Stroke};
```

Then write `stroke::Style`.